client/tailscale/localclient.go

```diff
@@ -114,6 +114,7 @@ func (lc *LocalClient) defaultDialer(ctx context.Context, network, addr string)
 //
 // DoLocalRequest may mutate the request to add Authorization headers.
 func (lc *LocalClient) DoLocalRequest(req *http.Request) (*http.Response, error) {
+	req.Header.Set("Tailscale-Cap", fmt.Sprint(int(tailcfg.CurrentCapabilityVersion)))
```

Comment:
would it be worth storing the string version to avoid stringification at every call?

Reply:
drop in the bucket I think. also IIRC all two byte int->string don't allocate

Reply:
Out of curiosity, is `fmt.Sprint` preferable to `strconv.Itoa`?

Reply:
strconv.Itoa would be a bit faster, actually. I can change.

I often use `fmt.Sprint` when I don't want to think about what the argument is.